api/configs/feature/__init__.py

```diff
@@ -136,6 +136,12 @@ class EndpointConfig(BaseSettings):
         default="",
     )
 
+    PROXY_FIX_MIDDLEWARE_ENABLED: bool = Field(
```

Comment:
Consider move this config from `EndpointConfig` class to `HttpConfig` class. It's not related to endpoint features.